ext/doc/generate_doc.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+php -d  extension=cassandra.so -d extension_dir=../modules/ generate_doc.php ..
```

Comment:
This is written assuming cwd is `ext/doc`, while the script in the dse driver assumes cwd is `ext`. I think this script should work no matter what cwd is. Something like:
```
#!/bin/sh
DIR=`dirname "$0"`
php -d extension=cassandra.so -d extension_dir="$DIR/../modules" "$DIR/generate_doc.php" $DIR/..
```